core/rendered_connection.ts

```diff
@@ -688,7 +692,8 @@ export class RenderedConnection
     return true;
   }
 
-  private findHighlightSvg(): SVGPathElement | null {
+  // TODO: Figure out how to make this private again.
```

Comment:
Please add a tsdoc with `@internal` so it doesn't get made part of our public API, and add same bug id from previous comment